CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -157,7 +157,11 @@ const char *_CFProcessPath(void) {
 
 #if DEPLOYMENT_TARGET_LINUX
 #include <unistd.h>
+#if __has_include(<syscall.h>)
 #include <syscall.h>
+#else
```

Comment:
won’t it error anyway? It doesn’t fail silently or anythhing anti-social. This just improves the odds. I can gate it if you want.